sonic_installer/bootloader/aboot.py

```diff
@@ -126,8 +126,8 @@ def set_next_image(self, image):
         return True
 
     def install_image(self, image_path):
-        run_command("/usr/bin/unzip -od /tmp %s boot0" % image_path)
-        run_command("swipath=%s target_path=/host sonic_upgrade=1 . /tmp/boot0" % image_path)
+        run_command(["/usr/bin/unzip", "-od", "/tmp", image_path, "boot0"])
+        run_command("swipath=%s target_path=/host sonic_upgrade=1 . /tmp/boot0" % image_path, shell=True)
```

Comment:
>. [](http://example.com/codeflow?start=66&length=1)

Is it possible to use string array as first argument? In your implementation, `is_list_of_strings` will fail if the first argument is string.